helloworld-mutual-ssl/configure-client-certs.cli

```diff
@@ -0,0 +1,18 @@
+# Configure a key-store in the Elytron subsystem. The path to the keystore file doesn’t actually have to exist yet.
```

Comment:
Just to make it more clear that this is just for the client's cert, maybe can reword to:

Configure the client's keystore. This will be used to generate the client's certificate. The path to the keystore file doesn’t actually have to exist yet

Reply:
fixed!